blockchain/chain/src/store/chain_store.rs

```diff
@@ -941,7 +943,7 @@ pub fn persist_block_messages<DB: Blockstore>(
 
     let mut bls_sigs = Vec::new();
     for msg in messages {
-        if msg.signature().signature_type() == SignatureType::BLS {
+        if msg.signature().signature_type() == SignatureType::BLS.into() {
```

Comment:
You can add something like:
```rust
pub fn signature_type(&self) -> SignatureType {
    self.0.signature_type().into()
}
```
to `Signature`.

Reply:
fixed

Reply:
Still need to propagate that change. See the lint errors.